### PR TITLE
mw_jwt updated

### PR DESCRIPTION
**Pull request by dencoded**
added changes to fix https://github.com/TykTechnologies/tyk/issues/1379

### CODE REVIEW

**Comment by buger** [comment]
Can you pls fix the tests, and update to use new test framework (mw_jwt_test.go already has examples of its usage).

**Comment by dencoded** [comment]
ok, working on tests to use new test instrumentation

File: helpers_test.go [hunk 1]
```diff
@@ -483,6 +500,13 @@ func buildAPI(apiGens ...func(spec *APISpec)) (specs []*APISpec) {
 			panic(err)
 		}
 
+		spec.Health = &DefaultHealthChecker{
```
Comment (buger) [inline helpers_test.go]: I assume this no longer needed since you migrated tests.

**Comment by buger** [comment]
@dencoded simplified tests a bit https://github.com/TykTechnologies/tyk/pull/1436/commits/af786916e6de2c3b64d7e59aecbb819630b3e635

**Pull request merged**